src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -27,17 +29,52 @@
     public CommandBox(CommandExecutor commandExecutor) {
         super(FXML);
         this.commandExecutor = commandExecutor;
+        this.history = CommandHistory.init();
```

Comment:
is there a need to use a static init? I think currently can just call constructor without args and create the empty arraylist inside the constructor